### ISSUE TITLE
Some delay in showing data at Site health's Status tab

### ISSUE THREAD

**Issue by YellowlimeNL** [LittleBigThing/carbon-footprint#11]
As the plugin needs some time to check whether website runs on renewable energy. Can we show a notification once that information is visible at Site health's status tab and link to it?

**Comment by LittleBigThing** [comment]
Good point: something to draw the attention of the user to the results.

To give some context:
Well, currently, in the worst-case scenario (homepage worse than 90% web pages tested via websitecarbon.com), a dashboard widget is shown for website admins with more information about the problem.
Otherwise, there is indeed no real trigger to look further (unless consulting Website Health directly). And there could be easy improvements for websites that are in a better segment of the test (better than the 10% worst :-)).

An idea for some light and friendly nudging is welcome! :-)

From the top of my head:
- a dashboard widget, similar to the worst-case but a bit less disruptive
- an admin notice (although we have enough of those for a lot of sites ;-))
- ...
